scalecube-configuration/src/test/java/resources/features/mpa-8092-Configuration service - Automatic testing.feature

```diff
@@ -1,122 +1,150 @@
 
-  - try to get the value you don't have permission to access and receive an error (a member can read and list (owner and admin are members))
 
-  - try to remove an entry you permission to remove and succeed with the change
-  - try to remove an entry you don't have permission to remove but have permission to view and receive an error
 
+  #Authorization:
+  #
+  #- try to add an entry you have permission to add and succeed with the change
+  #- try to add an entry you don't have permission to add but have permission to view and receive an error
+  #
+  #- try to edit a value you permission to edit and succeed with the change
+  #- try to edit a value you don't have permission to edit but have permission to view and receive an error
+  #
+  #- try to get the value you don't have permission to access and receive an error (a member can read and list (owner and admin are members))
+  #
+  #- try to remove an entry you permission to remove and succeed with the change
+  #- try to remove an entry you don't have permission to remove but have permission to view and receive an error
 
-  As a user I would like to manage instruments instances in the configuration service
 
-  For example:
 
-  - add a new instrument instance with the same structure
-  - edit the following metadata for instrument instance: DecimalPrecision, Rounding, name, rounding
-  - remove and existing instrument instance configuration
+  #CREATE REPO
 
-  #_____________________________________________________CRUD____________________________________________________________
+  #__________________________________________________POSITIVE___________________________________________________________
 
-  #MPA-7103 (#1) - SHOULD WE RETURN THE REPO INFO INSTEAD OF PlAIN ACKNOWLEDGEMENT?
+  #MPA-8092 (#1)
   Scenario: Successful Repo creation
     Given a user have got a valid "token" (API key) with assigned "owner" role
     When this user requested to create the "repository" with "specified" name
     Then new "repository" should be created and stored in DB
 
 
-  #MPA-7103 (#1.1)
-  Scenario: Fail to create the Repo upon access permission is restricted
-    Given the users have got the valid "tokens" (API key) with "admin" and "member" assigned roles
-    When each of these users requested to create the "repository" with "specified" name
-    Then any of repositories shouldn't be created and each of the user should get an error message: "role: "userId" not in role Owner"
-
+  #__________________________________________________NEGATIVE___________________________________________________________
 
-  #MPA-7103 (#1.2)
-  Scenario: Fail to create the Repo upon a "token" is invalid (expired)
-    Given a user have got an invalid "token" (API key)
-    When this user requested to create the "repository" with "specified" name
-    Then new "repository" shouldn't be created and the user should get an error message: "Token verification failed"
+  #MPA-8092 (#2)
```

Comment:
this means it is implemented?

Reply:
As you said before non of the tests wasn’t implemented.
So this file is not the exeption so each of these tests which are specified in this feture file not implemented.
I started ti update existing ones which I already pushed up to mpa-7103. 
But during the implementation Aharon closed his pull request under these tests i guess due to the spots in the business logic.

Reply:
As you said before non of the tests wasn’t implemented.
So this file is not the exeption so each of these tests which are specified in this feture file not implemented.
I started ti update existing ones which I already pushed up to mpa-7103. 
But during the implementation Aharon closed his pull request under these tests i guess due to the spots in the business logic.

Reply:
I am goong to proceed with tests and discuss them with the team, i guess potentially with Dmytro and Gabriel, I wanted to involve Assaf to the discussion also